models/labels/arb_traders/labels_arb_traders_ethereum.sql

```diff
@@ -0,0 +1,33 @@
+{{config(alias='arb_traders_ethereum')}}
```

Comment:
This model needs to be moved under the models/labels/arb_trades/ethereum directory